artist.py

```diff
@@ -0,0 +1,72 @@
+import pygame
+import os
+from common_constants import Vector
+
+# in charge of drawing Game objects
+class Artist:
+    def __init__(self, game):
+        self.game = game
+
+        # pygame setup
+        os.environ['SDL_VIDEO_WINDOW_POS'] = "%d,%d" % (0,0) # locate the window in the upper left
+        pygame.init()
+        pygame.display.set_caption(f'{type(game).__name__} Reduction')
```

Comment:
Not sure if this is the right place for this...is Artist just for rendering the contents of the screen? And we're leaving NP-Gravity to handle the "metagame" (i.e. pygame init, game loop, window title, stuff like that)?

Reply:
i think this is ok as the os.environ and set_caption calls are graphics related and i'm conceptualizing Artist as handling everything related to graphics output, while np-gravity will specialize in taking user input and sending it where it needs to go. the pygame.init() call is debatable, but since it would be the only pygame call in np-gravity and we do want to ensure pygame has been initialized before we start making calls to it in Artist, i'm in favor of leaving it here